### ISSUE TITLE
Adding elements above the ListView in tab-view

### ISSUE THREAD

**Issue by adremalin** [react-native-community/react-native-tab-view#120]
Hi!

I am trying to add a View element above the ListView in a tab-view, but get following error message:

```
Element type is invalid: expected a string for built in components or a class/function (for composite components) but got: undefined. Check the render method of 'TabViewTranstitioner'
```

What is the problem here? 

Thanks!

**Comment by satya164** [comment]
Can you post your code please? It's impossible to help without seeing the code.

**Comment by adremalin** [comment]
**Yes! I have the exact code from the example and just pasting in the following snippet in the example code below:**
          <View>
                 <Text style={{ color: 'white' }}>hi</Text>
                 <Button> Save </Button>
          </View>

**_renderScene method from the example code:**

_renderScene = ({ route }) => {
    switch (route.key) {
    case '1':
      return (<View style={[styles.page, { backgroundColor: 'white' }]}>
                <View>
                 <Text style={{ color: 'white' }}>hi</Text>
                 <Button> Save </Button>
                 </View>
                <ListViewExample />
             </View>);
    case '2':
      return (<View style={[styles.page, { backgroundColor: 'white' }]}>
                <ListViewExample />
             </View>);
    case '3':
      return (<View style={[styles.page, { backgroundColor: 'white' }]}>
                <ListViewExample />
              </View>);
    case '4':
      return (<View style={[styles.page, { backgroundColor: 'white' }]}>
                <ListViewExample />
              </View>);
    default:
      return null;
    }
  };

**Comment by adremalin** [comment]
Hey @satya164, you found and answer for this? :)

**Comment by CharlesMangwa** [comment]
Hi @adremalin! Here is a working example based on the code you've provided: https://sketch.expo.io/HkM8MIhix.

**Comment by CharlesMangwa** [comment]
Closed #120.
